css-overflow-5/Overview.bs

```diff
@@ -155,6 +155,28 @@ its content will be integrated into this specification,
 which will then replace it.
 Until then, this specification only contains additions and extensions to level 4.
 
+<h2 id="overflow-concepts">
+Overflow Concepts and Terminology</h2>
+
+Issue: Copy [[css-overflow-3#overflow-concepts|Level 3 content]] when final.
+
+<h3 id="scrolling">Scrolling overflow</h3>
+
+The following is added to the concepts in scrolling overflow:
+
+Every [=scroll container=] maintains a <dfn export>current scroll target</dfn>.
+It is initially <code>null</code>,
+and is reset to <code>null</code> after any scrolling operations in that [=scroll container=]
+initiated by the user,
+or any script initiated operations that do not have a target,
+or when the target is removed from the document.
+Scrolling operations in the [=scroll container=] with a target Element or PseudoElement,
+set the [=current scroll target=] to that target Element or PseudoElement.
+
+Issue: Setting the current scroll target should be defined in how to <a>scroll a target into view</a>.
+
+Issue: Use the current scroll target as an <a>anchor priority candidate</a> for scroll anchoring.
```

Comment:
I was trying to link this to the definition in css-scroll-anchoring, but it doesn't seem to be working. I'm sure I missed something simple.

Reply:
Scroll Anchoring defines the term as just "priority candidates", which isn't great. I'll go fix that spec, it should really be exporting as "anchor priority candidate"; this will start automatically linking then.

Reply:
Great, tested and it is now linking.